core/templates/components/common-layout-directives/navigation-bars/top-navigation-bar.directive.html

```diff
@@ -92,12 +92,13 @@
                  translate="I18N_TOPNAV_GET_STARTED">
               </a>
             </li>
+            <!-- TODO(#12070): Commented out until new pages/modals are made.
```

Comment:
Generally we just file the issue, and dont really keep commented out code in the codebase. But I think this is fine for this situation?

Reply:
done.